doc/nrf/samples.rst

```diff
@@ -3,10 +3,21 @@
 Samples
 #######
 
+`Zephyr`_ contains a variety of application samples and demos.
+Documentation for those is available in the :ref:`zephyr:samples-and-demos` section.
+
 For very simple samples, see the :ref:`zephyr:basic-sample` in Zephyr.
 Those samples are a good starting point for understanding how to put together your own application.
```

Comment:
I'm not a fan of starting the section in the NCS with so many references to Zephyr. I would suggest something like this:
```
The |NCS| provides samples that specifically target Nordic Semiconductor devices and show how to implement typical use cases with Nordic libraries and drivers.

Samples showcase a single feature or library, while applications include a variety of libraries to implement a specific use case.

The samples in the |NCS| are compatible with the variety of application samples and demos available in Zephyr.
Documentation for those is available in Zephyr's :ref:`zephyr:samples-and-demos` section.

For very simple samples, see the :ref:`zephyr:basic-sample` in Zephyr.
Those samples are a good starting point for understanding how to put together your own application.

[remaining lines unchanged]
```

Reply:
Thanks Greg. Used this.